browser-test/src/question_lifecycle.test.ts

```diff
@@ -255,4 +255,36 @@ describe('normal question lifecycle', () => {
       ),
     ).toBeTruthy()
   })
+
+  it('redirects to draft question when trying to edit original question', async () => {
+    const {page} = await startSession()
+    page.setDefaultTimeout(4000)
+    await loginAsAdmin(page)
+
+    const adminQuestions = new AdminQuestions(page)
+    const adminPrograms = new AdminPrograms(page)
+
+    await adminQuestions.gotoAdminQuestionsPage()
+    await adminQuestions.addNameQuestion({questionName: 'name-q'})
+
+    const programName = 'test program'
+    await adminPrograms.addProgram(programName)
+    await adminPrograms.publishProgram(programName)
+
+    // Update the question to create new draft version.
+    await adminQuestions.gotoQuestionNewVersionPage('name-q')
+    // Save edit url as later we'll use it to imitate user having multiple tabs open.
+    const editUrl = page.url()
```

Comment:
Should we navigate to the question page and grab the URL prior to going to the new version page? As this currently reads, it makes me think we're saving off the URL of the new question version rather than the old question version.

Reply:
Hmm. Assuming that published question id is `10` then "new version url" will be `/admin/questions/10/edit` which is the same as "old question edit url".

Reply:
Right you are. That's not entirely obvious. Would you mind updating the comment to reflect this, maybe something like:
The ID in the URL after clicking new version corresponds to the active question form. After a draft is created, the ID will reflect the newly created draft version.